app/controllers/administrate/application_controller.rb

```diff
@@ -40,7 +41,7 @@ def edit
     end
 
     def create
-      resource = new_resource(resource_params)
+      @new_resource = resource = new_resource(resource_params)
```

Comment:
And same thing here:

```suggestion
      @resource = resource = new_resource(resource_params)
```